documentation/source/conf.py

```diff
@@ -69,10 +71,10 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
-# The short X.Y version.
-version = '1.1'
 # The full version, including alpha/beta/rc tags.
-release = '1.1'
+release = cocotb.__version__
+# The short X.Y version.
+version = '.'.join(release.split('.')[:2])
```

Comment:
Handwritten version parsing always makes me nervous. In this case, we get slightly inconsistent behavior for "1.2rc1" vs "1.2.0rc1". Looking through the code, it seems we could get rid of the version vs release distinction and always use the full "release"?

Reply:
> Handwritten version parsing always makes me nervous.

That's fair, but the cost of getting this wrong is a worst a weird string somewhere in the generated docs. Also, I think it would be rather foolish of us to change the number of components in our version.

> Looking through the code,

Which code? Sphinx itself?

---

I could switch this to:
```
from distutils.version import LooseVersion  # note: builtin
version = '.'.join(LooseVersion(release).version[:2]))
```

> and always use the full "release"

I'm ok.0 with this if you are.0 :)